uiowa/uiowa
# [ P2LB ] V2 Revert action

Tracks https://github.com/uiowa/uiowa/issues/6707

Would love to have this only show on the converted form



# How to test

```
ddev blt ds --site brand.uiowa.edu && ddev drush @brand.local config-split:activate p2lb -y && ddev drush @brand.local cim -y && ddev drush @brand.local uli admin/content/sitenow-converter
```
- Convert the "Web" page
- Check revisions to make sure that the correct revision was created with a note about the conversion
- View the latest revision tab and make sure that the header is not duplicated
- View the layout tab and ensure that the paragraph sections do not display
- Revert the page
- Make sure that everything is back to normal
- Review code changes and spot check other P2LB functionality to make sure the refactor didn't break anything
